lib/autokey/scripting/window.py

```diff
@@ -309,24 +314,28 @@ def get_window_list(self, filter_desktop=None):
         C{desktop} is the number of which desktop (sometimes called workspaces) the item appears upon.
 
         C{hostname} is the hostname of your computer (I'm not sure why this is included as it seems to be the same for everything I have seen)
```

Comment:
This is inappropriate in a docstring comment. Stick it outside the docstring.

As to why it is included: No idea